src/screens/wallet/SettingsScreen.tsx

```diff
@@ -121,8 +121,10 @@ const SettingsScreen = (props: ScreenProps<'SettingsScreen'>) => {
         true,
         '',
         () => {
-          setLoading(undefined);
-          goBack();
+          setTimeout(() => {
+            setLoading(undefined);
```

Comment:
In that case, if loading could be a boolean, like true or false, would be better?
Thinking about the moment I'm reading the code, if I see setLoading(false) I could understand it better.

Reply:
that's a good observation. I'll fix it in a new pr